toolchain/check/testdata/class/generic_method.carbon

```diff
@@ -141,12 +141,14 @@ fn Class(T:! type).F[self: Self](n: T) {}
 // CHECK:STDOUT:   %complete_type.loc14_1.2 => constants.%complete_type
 // CHECK:STDOUT: }
 // CHECK:STDOUT:
+// CHECK:STDOUT: specific @Class(%T.loc11_13.2) {}
+// CHECK:STDOUT:
+// CHECK:STDOUT: specific @Class(@F.%T.loc13) {}
+// CHECK:STDOUT:
 // CHECK:STDOUT: specific @F(constants.%T) {
 // CHECK:STDOUT:   %T.loc13 => constants.%T
 // CHECK:STDOUT:   %Class => constants.%Class
 // CHECK:STDOUT: }
 // CHECK:STDOUT:
-// CHECK:STDOUT: specific @Class(@F.%T.loc13) {}
-// CHECK:STDOUT:
-// CHECK:STDOUT: specific @Class(%T.loc11_13.2) {}
+// CHECK:STDOUT: specific @Class(<unexpected>.inst66.loc16_28) {}
```

Comment:
And here. I wonder if it's related to the `@Class` disappearing on L53? Or maybe DiscardGeneric?